plugins/jreleaser-gradle-plugin/src/main/groovy/org/jreleaser/gradle/plugin/internal/dsl/GitlabImpl.groovy

```diff
@@ -61,6 +64,7 @@ class GitlabImpl extends AbstractGitService implements Gitlab {
         if (changelog.isSet()) service.changelog = changelog.toModel()
         if (milestone.isSet()) service.milestone = milestone.toModel()
         if (commitAuthor.isSet()) service.commitAuthor = commitAuthor.toModel()
+        service.identifier = identifier
```

Comment:
Good. However this call might produce an error if the value has not been initialized. There are a couple of ways to prevent that, the rest of the code does it like this `if (identifier.present) service.identifier = identifier.get()`.